docs/features/pouch_with_diskquota.md

```diff
@@ -47,25 +47,37 @@ Both two dimensions are covered in PouchContainer's diskquota.
+* rule1: `--disk-quota=10GB` : maps container rootfs and all potential volumes binded inside;
+* rule2: `--disk-quota=/abc=10GB` : absolute path matching, maps only mount point `/abc` has been limited, container rootfs and any volume haven't been limited;
+* rule3: `--disk-quota=/&/abc=10G` : shared size matching, maps container rootfs and mount point `/abc` have been limited, and their total block size are 10G;
+* rule4: `--disk-quota=.*=10G`: regular expression matching, maps container rootfs and each mount points have been limited to 10G independently.
 
-* `.*=10GB` maps container rootfs and all potential volumes binded inside;
-* `/=10GB` : maps only container rootfs without any volume binded directory in container;
-* `/=10GB,/a=10GB`: the front part maps container rootfs, and the back one maps the volume which is binded to directory `/a` inside container.
+Flag `--quota-id string` is used to pick an existent quota ID to specify the newly input disk quota. The input type is `string` as well.
+
+There are three ways to identify the input format:
 
-Flag `--quota-id` is used to pick an existent quota ID to specify the newly input disk quota. The input type is `string` as well. If input `quota-id` is less than 0, pouchd will automatically generate one brand new quotaid and return this ID. If input `quota-id` is 0, pouchd will not set quotaid. If `quota-id` is larger than 0, pouchd will use the input quota ID to set disk quota.
+* `--quota-id=-1` : it means pouch daemon will assign a separate available quota id to each mount point, include container rootfs;
+* `--quota-id=0` or --quota-id="" or haven't set: it means pouch daemon don't assign quota id;
+* `--quota-id=16777216` or more than `16777216`, it means using the specified quota id to set quota.
 
-> valid `quota-id` which is larger than 0 is only used in `upgrade` interface. In this scenario of triggering `upgrade` interface, pouchd will remove the old container and use the new image to take place of old container's image, and create a new container which should inherit the original diskquota. Then user can pass an original `quota-id` of original container to take effect on newly created container.
+> 1. `rule1` can't use with `rule2/rule3/rule4` together;
+> 2. if it has set quota-id isn't null or `0`, `--disk-quota` only can been set with one rule, or means the length of disk quota slice is `1`.
+> 3. `rule3` use `&` link with directory that must be absolute path, can't be regular expression with `rule4`;
+> 4. no special characters(`* & ;`) can exist in all mount points
+> 5. valid `quota-id` which is larger than `16777216` is only used in `upgrade` interface. In this scenario of triggering `upgrade` interface, pouchd will remove the old container and use the new image to take place of old container's image, and create a new container which should inherit the original diskquota. Then user can pass an original `quota-id` of original container to take effect on newly created container.
```

Comment:
Using a given quota-id is a common scene, not only in 'upgrade' interface.

Reply:
modified